src/Symfony/Component/HttpKernel/DataCollector/DataCollector.php

```diff
@@ -41,14 +50,42 @@ public function unserialize($data)
     }
 
     /**
+     * Converts the variable into a serializable Data instance.
+     *
+     * This array can be displayed in the template using
+     * the VarDumper component.
+     *
+     * @param mixed $var
+     *
+     * @return Data
+     */
+    protected function cloneVar($var)
+    {
+        if (null === $this->cloner) {
+            $this->cloner = new VarCloner();
```

Comment:
maybe through a setter that is used for all services tagged with `data_collector`?